src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesTransportFactory.php

```diff
@@ -31,6 +32,10 @@ public function create(Dsn $dsn): TransportInterface
         $port = $dsn->getPort();
 
         if ('ses+api' === $scheme) {
+            if (!class_exists(\SimpleXMLElement::class)) {
+                throw new LogicException(sprintf('Cannot use "%s". Make sure you have "ext-simplexml" installed and enabled.', SesApiTransport::class));
```

Comment:
Beware when merging this in 5.1 @symfony/mergers : The message should only applied in `if (!class_exists(SesClient::class)) {` block